src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator/config.jelly

```diff
@@ -1,30 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
-    <f:entry title="${%Owner}" field="repoOwner">
-        <f:textbox/>
-    </f:entry>
-    <f:entry title="${%Scan credentials}" field="credentialsId">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials" xmlns:scm="/jenkins/scm/api/form">
+  <j:choose>
+    <j:when test="${descriptor.serverUrlSelectable}">
+      <f:entry title="${%Server}" field="serverUrl">
+        <f:select/>
+      </f:entry>
+    </j:when>
+    <j:otherwise>
+      <f:invisibleEntry>
+        <f:select field="serverUrl"/>
+      </f:invisibleEntry>
+    </j:otherwise>
+  </j:choose>
+  <f:entry title="${%Credentials}" field="credentialsId">
         <c:select/>
     </f:entry>
-    <f:entry title="${%Repository name pattern}" field="pattern">
-        <f:textbox default=".*"/>
-    </f:entry>
-    <f:entry field="autoRegisterHooks">
-        <f:checkbox title="${%Auto-register webhooks}" />
-    </f:entry>
-    <f:advanced>
-        <f:entry title="${%Include branches}" field="includes">
-            <f:textbox default="*"/>
-        </f:entry>
-        <f:entry title="${%Exclude branches}" field="excludes">
-            <f:textbox/>
-        </f:entry>
-        <f:entry title="${%Checkout credentials}" field="checkoutCredentialsId">
-            <c:select default="${descriptor.SAME}"/>
-        </f:entry>
-        <f:entry title="${%Bitbucket Server URL}" field="bitbucketServerUrl">
-          <f:textbox />
-        </f:entry>
-    </f:advanced>
+  <f:entry title="${%Owner}" field="repoOwner">
+    <f:textbox/>
+  </f:entry>
+  <f:entry title="${%Behaviours}" field="traits">
```

Comment:
`SCMBehaviour` as a class name was too generic. `traits` in the UI was deemed too confusing to end users. 🙄